src/sdk/models/auditCategories.ts

```diff
@@ -1,7 +1,9 @@
 import { IEntityIdentifier } from './base';
 
 export interface AuditCategory extends IEntityIdentifier<string> {
+	id?: string;
```

Comment:
This shouldn't be necessary as the entity derives from `IEntityIdentifier` already.

Reply:
rolled back